tests/graph_test.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2019 IBM. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""
+Tests for graph.py in the GraphDef Editor
+"""
+
+import unittest
+import tensorflow as tf
+import numpy as np
+import shutil
+import tempfile
+
+
+import graph_def_editor as gde
+
+class GraphTest(unittest.TestCase):
+
+  def setUp(self):
+    # Create a temporary directory for SavedModel files.
+    self.temp_dir = tempfile.mkdtemp()
+
+  def tearDown(self):
+    # Remove the directory after the test.
+    # Comment out this line to prevent deleting temps.
+    shutil.rmtree(self.temp_dir)
+    pass
```

Comment:
don't need the `pass`

Reply:
You do if you comment out the previous line ;-) I'll add a comment to that effect.

Reply:
Ah, I see. I totally missed the comment :)